.github/workflows/ci-unix-shared.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       uses: actions/cache@v3
       with:
         path: ext
-        key: ${{ runner.os }}-${{ hashFiles('ext/*.cmd') }}
+        key: ${{ runner.os }}-shared-${{ hashFiles('ext/*.cmd') }}
```

Comment:
I hope that the `hashFiles('ext/*.cmd')` call uses the contents of the `ext/*.cmd` files before we modify them. It seems to this way.